DFC.Integration.AVFeed/DFC.Integration.AVfeed.ClearRecycleBinFunction/ClearRecycleBin.cs

```diff
@@ -0,0 +1,33 @@
+﻿using DFC.Integration.AVFeed.Data.Interfaces;
+using DFC.Integration.AVFeed.Data.Models;
+using System.Threading.Tasks;
+
+
+namespace DFC.Integration.AVFeed.Function.ClearRecycleBin
+{
+    public class ClearRecycleBin : IClearRecycleBin
+    {
+        private readonly ICustomApiContextService customApiContextService;
+        private readonly IApplicationLogger logger;
+        private readonly IAuditService auditService;
+        private readonly int numberToClear = 20;
```

Comment:
Are we hard coding this? Remember will probably have around 1500 records to delete per day.  Can we put this in a config? on my local box it could handle about 1000 - 1500 before it would timeout..

Reply:
Now configurable, done in batches over a number of time.